packages/provider-url-service/firebase-alfajores-env-config.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+firebase functions:config:set \
+transak.public_key=$(grep TRANSAK_PUBLIC_KEY_STAGING .env | cut -d '=' -f 2-) \
+transak.private_key=$(grep TRANSAK_PRIVATE_KEY_STAGING .env | cut -d '=' -f 2-) \
+ramp.public_key=$(grep RAMP_PUBLIC_KEY_STAGING .env | cut -d '=' -f 2-) \
+moonpay.public_key=$(grep MOONPAY_PUBLIC_KEY_STAGING .env | cut -d '=' -f 2-) \
+moonpay.private_key=$(grep MOONPAY_PRIVATE_KEY_STAGING .env | cut -d '=' -f 2-) \
```

Comment:
Leaving this somewhat manual for now because I couldn't figure out how to properly nest provider specific variables (e.g., do `moonpay.public_key` instead of `moonpay_public_key`) and didn't want to pollute the env namesapce